src/bde_copy.cpp

```diff
@@ -1582,6 +1599,12 @@ bool read_args( char *image, int argc, char *argv[] )
         printf("Invalid dataset argument \"%s\" - must by YYYYMMDDHHMMSS\n",dataset);
         argsok = false;
     }
+    
+    if( use_gzip && output_stdout )
+    {
+        printf("Cannot use -z option with output to standard output\n");
```

Comment:
errors should be printed to stderr, not stdout

Reply:
I changed this to print to stderr, in doing this I also updated the other errors so that they also print to stderr.